src/docfx/Errors.cs

```diff
@@ -454,6 +455,13 @@ public static Error OverDateRange(SourceInfo source, string name, TimeSpan? rela
         public static Error FieldDeprecated(SourceInfo source, string name, string replacedBy)
             => new Error(ErrorLevel.Warning, "field-deprecated", $"Deprecated field: '{name}'{(string.IsNullOrEmpty(replacedBy) ? "." : $", use '{replacedBy}' instead")}", source);
 
+        /// <summary>
+        /// The values of the two fields do not match.
+        /// </summary>
+        /// Behavior: ✔️ Message: ❌
+        public static Error ValuesNotMatch(SourceInfo source, string name, string value, string dependentFieldName, string dependentFieldValue)
+            => new Error(ErrorLevel.Warning, "values-not-match", $"Invalid value for {name}: '{value}' is not valid with '{dependentFieldName}' value '{dependentFieldValue}'", source);
```

Comment:
consider pass the accepted values just like `Enum` validation. Also, `Error.Join` method currently does not show more than 5 items, could you help update that method to append `...` when the item count is larger than 5 to indicate there are more to values than displayed?

Reply:
okok